### PR TITLE
fix(tools): get eslint more properly configured

### PR DESCRIPTION
**Pull request by DerekNonGeneric**

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+!**/.*.*
+!.**/**/*
+
 .git/
 **/_site/
 **/node_modules/
 /vendor/**
 package.json
+
+LICENSE
```
Comment (DerekNonGeneric) [inline .prettierignore:13]:
```suggestion
LICENSE

```

**Review by DerekNonGeneric — COMMENTED** [review]
I am not sure, but this seems to be incorrect.

I am unsure who else is using VS Code.

/cc @jorgebucaran @yuvilio are y'all using VSCode?

File: .vscode/settings.json [hunk 2]
```diff
@@ -41,23 +39,7 @@
...
-  },
-
-  "terminal.integrated.automationProfile.linux": {
-    "path": "fish",
-    "icon": "terminal"
-  },
-
-  "terminal.integrated.env.linux": {
-    // TODO: Determine whether defining env-vars here would be preferable.
-  },
```
Comment (DerekNonGeneric) [inline .vscode/settings.json:57]: All these fields were complaining about going to either be deprecated soon or currently were deprecated&hellip;

File: .vscode/settings.json [hunk 3]
```diff
@@ -152,11 +135,9 @@
   // Whether the extension contributes a lint task to lint a whole workspace.
   "eslint.lintTask.enable": true,
 
+  "eslint.workingDirectories": ["./"],
+
   // Avoids having to globally install ESLint.
   // TODO(DerekNonGeneric): See about switching to pnpm!
```
Comment (DerekNonGeneric) [inline .vscode/settings.json:141]:
```suggestion
  // Avoids having to globally install ESLint.
  // TODO(DerekNonGeneric): See about switching to pnpm!
```

Actually, maybe not in this particular repo&hellip;

File: .vscode/settings.json (hunk 2 above)
Comment (jorgebucaran) [reply .vscode/settings.json:57]: FWIW, I've never used this and never had trouble with Fish and VSCode built-in's terminal.

**Review by jorgebucaran — APPROVED** [review]

File: .eslintrc.json [hunk 4]
```diff
@@ -1,43 +1,117 @@
...
+      "jsx": false
     },
+    "project": ["./tsconfig.eslint.json"],
     "sourceType": "module"
   },
+  "plugins": [
+    "import",
+    "json-schema-validator",
+    "unicorn",
+    "import",
```
Comment (DerekNonGeneric) [inline .eslintrc.json:23]: Repeated `import` here by accident.. going for a second round soon.

File: .eslintrc.json [hunk 5]
```diff
@@ -1,43 +1,117 @@
...
-        "project": "./tsconfig.eslint.json"
+        "project": ["./tsconfig.eslint.json"],
+        "warnOnUnsupportedTypeScriptVersion": false
       },
+      "plugins": [
+        "@typescript-eslint",
+        "import",
+        "unicorn",
+        "import",
+        "wix-editor",
```
Comment (DerekNonGeneric) [inline .eslintrc.json:97]: Same as above..

File: .eslintrc.json [hunk 6]
```diff
@@ -1,43 +1,117 @@
...
+        "prettier",
+        "simple-import-sort",
+        "promise",
+        "prettier"
```
Comment (DerekNonGeneric) [inline .eslintrc.json:101]: Another repeated import, wonder if this would be caught w/ JSON schema

File: .eslintrc.json [hunk 7]
```diff
@@ -1,43 +1,117 @@
...
+      ],
       "extends": [
         "eslint:recommended",
         "plugin:import/recommended",
-        "plugin:import/typescript",
```
Comment (DerekNonGeneric) [inline .eslintrc.json:36]: This should have stayed i think.. 🤔

**Pull request merged**